NEWS.md

```diff
@@ -86,6 +86,8 @@ New library functions
 * New function `bitreverse` for reversing the order of bits in a fixed-width integer ([#34791]).
 * New function `bitrotate(x, k)` for rotating the bits in a fixed-width integer ([#33937]).
 * One argument methods `startswith(x)` and `endswith(x)` have been added, returning partially-applied versions of the functions, similar to existing methods like `isequal(x)` ([#33193]).
+* New function `isgreater(a, b)` defines a descending total order where unorderable values and missing are ordered smaller than any regular value.
```

Comment:
This sentence sounds contradictory to me. It says `isgreater` orders unorderable values. How about phrasing it like this?:

> New function `isgreater(a, b)` defines a descending total order that is compatible with the pass-through behavior of `min`; i.e., non-`NaN` numbers are greater than `NaN` and non-`missing` values are greater than `missing`.

I think it's better to rephrase the docstring with a similar wording.

If you want to use the word "unorderable" instead, maybe "(`<`, `=`)-unorderable" is better?

Reply:
`isless` uses the phrase "values that are normally unordered, such as `NaN`", which I had intended to imitate, but forgot :)

Reply:
Updated NEWS and the docstring.